samples/function_chaining_custom_status/DurableActivity/__init__.py

```diff
@@ -0,0 +1,27 @@
+import logging
+
+def main(name: str) -> str:

```

Comment:
Can we change `name` to `key`? The `name` parameter seems more appropiate to a `hello world` function